### DISCUSSION TITLE
vercel/.editorconfig

### DISCUSSION THREAD

**Discussion by AnAtOlIy1994** [vercel/vercel#8803] [Show and tell]
![IMG_20221030_023207](https://user-images.githubusercontent.com/116979899/198856711-de9ec0b6-7e8b-41ad-b39e-ca28e2eda894.jpg

**Comment by amyegan** [comment]
Hi @AnAtOlIy1994! Can you share more information about the error you're seeing and what steps I can take to reproduce it for debugging?
